packages/component-model-survey-response/typeDefs.graphqls

```diff
@@ -0,0 +1,23 @@
+type SurveyResponse implements Object {
+  id: ID!
+  created: DateTime!
+  updated: DateTime
+  surveyId: ID!
+  manuscriptId: ID!
+  repsonse: SurveyQuestionsAndAnswers
+}
```

Comment:
Wait so does this supercede the graphql defined in `component-submission`? If so why?

Reply:
`component-submission` would return `SurveyResponse` from its `typeDefs.graphql` defined `Queries` and `Mutations`. The object is defined in the model component, the queries and mutations are defined in the context component